File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.js
function solution(s) {
    var answer = '';
    const arr = s.split(" ").map(w => w.toLowerCase())
    
    
    for (let i = 0; i < arr.length; i++){
        if (arr[i] === "")
            continue
        arr[i] = arr[i][0].toUpperCase() + arr[i].slice(1)
        
    }
    
    return arr.join(" ");
}

File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.java
import java.util.*;

class Solution {
    public String solution(String s) {
        String answer = "";
        
        String[] arr = s.split(" ",-1);
        
        for (int i = 0; i < arr.length; i++){
            if (arr[i].length() == 0){
                continue;
            }
            arr[i] = arr[i].substring(0,1).toUpperCase() + arr[i].substring(1).toLowerCase();
        }
        
        answer = String.join(" ",arr);
        
        return answer;
    }
}